packages/plugin-react-native-remote-spans/lib/index.ts

```diff
@@ -1,32 +1,70 @@
 import { TurboModuleRegistry } from 'react-native'
-import type { Spec } from './NativeBugsnagRemoteSpans'
-import type { ParentContext } from '@bugsnag/core-performance'
+import { SpanQuery } from '@bugsnag/core-performance/lib/span-control-provider'
+import type { SpanUpdateTransaction, Spec } from './NativeBugsnagRemoteSpans'
+import type { ParentContext, SpanAttribute, Time } from '@bugsnag/core-performance'
+import type { SpanControlProvider } from '@bugsnag/core-performance/lib/span-control-provider'
```

Comment:
`SpanQuery and `SpanControlProvider` should be exported from `@bugsnag/core-performance`